sample/spec/spec_helper.rb

```diff
@@ -12,6 +12,7 @@
 require 'rspec/rails'
 require 'ffaker'
 require 'spree_sample'
+require 'factory_girl'
```

Comment:
This will generate a warning and I don't think it is necessary after a rebase (the Gemfile will cause factory_bot to be required).

It turns out we also don't need factories at all for this test suite, so I have removed them in #2330